presto-hive/src/main/java/io/prestosql/plugin/hive/s3/ConfigurationInitializer.java

```diff
@@ -15,7 +15,7 @@
 
 import org.apache.hadoop.conf.Configuration;
 
-public interface S3ConfigurationUpdater
+public interface ConfigurationInitializer
```

Comment:
How is `ConfiguratonInitializer` similar to `DynamicConfigurationProvider`?
Could we just use `DynamicConfigurationProvider`?

Probably we should rename `DynamicConfigurationProvider` to `DynamicConfigurationUpdater` if it's still needed.

Reply:
Initializer is used only once, while `DynamicConfigurationProvider` is used to provide session based configuration (which is based on the one that was previously initialized).